File: Lista 1 - programação/Q2.c
//2) Ler três números, possíveis lados de um triângulo, e imprimir a classificação segundo os lados (se for o  caso).

#include <stdio.h>

int main () {

int lado1, lado2, lado3;
    printf("Digite o valor dos tres lados do triangulo: ");
    scanf ("%d %d %d", &lado1, &lado2, &lado3);

    if (lado1 == lado2 == lado3 == lado1)
        printf ("O triangulo e Equilatero");

    else if (lado1 == lado2)
        printf("O triangulo e Isoceles");
            else if (lado2 == lado3)
                printf("O triangulo e Isoceles");
                else if (lado3 == lado1)
                    printf("O triangulo e Isoceles");

    else
        printf("O triangulo e Escaleno");
return 0;
}

File: Lista 1 - programação/Q2.py
#2) Ler três números, possíveis lados de um triângulo, e imprimir a classificação segundo os lados (se for o  caso)

l1 = int(input("Digite os lados do triângulo "))
l2 = int(input("Digite os lados do triângulo "))
l3 = int(input("Digite os lados do triângulo "))

m = (l1 + l2 + l3) - max(l1, l2, l3)

if (max(l1, l2, l3)) > m:
    print("O triângulo não existe")

else:
    if l1 != l2 and l2 != l3 and l3 != l1:
        print("O triângulo é Escaleno")
    elif l1 == l2 == l3:
        print("O triângulo é Equilatero")
    else:
        print("O triângulo é Isoceles")
